cvat/apps/api_tokens/admin.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (C) CVAT.ai Corporation
+#
+# SPDX-License-Identifier: MIT
+
+from django.contrib import admin
+from rest_framework_api_key.admin import APIKey, APIKeyAdmin
+
+from .models import ApiToken
+
+
+class ApiTokenAdmin(APIKeyAdmin):
+    list_display = (
+        ["id", "owner"]
+        + list(APIKeyAdmin.list_display)
+        + [
+            "_is_stale",
```

Comment:
Could you leave a link to https://code.djangoproject.com/ticket/31558 here as a reminder? It looks like this bug was fixed in Django 5.0, so we'll eventually be able to remove the `_is_stale` function.